packages/core/src/components/Dashboard/index.tsx

```diff
@@ -76,7 +78,7 @@ export function Dashboard({
         }
 
         setColors(data.tokens.colorData ?? [])
-        setTypography(data.tokens.typography)
```

Comment:
In main, the Dashboard index file was refactored. so this needs to be updated with what's on main